docs-ref-conceptual/param-persist-howto.md

```diff
@@ -23,15 +23,15 @@ Configuration values used by the CLI are evaluated in the following precedence,
 1. Environment variables
 1. Values in the configuration file or set with `az config`
 
-[Install the Azure CLI](install-azure-cli.md) or open [Azure Cloud Shell](https://shell.azure.com) to run the scripts in this article.  If you are using a local install of the Azure CLI, version 2.12.0 or later is needed to run `az config param-persist` commands.  Run [az version](/cli/azure/reference-index#az_version) to find the version and dependent libraries that are installed. To upgrade to the latest version, run [az upgrade](/cli/azure/reference-index#az_upgrade).  Azure Cloud Shell always has the latest version of the Azure CLI.
+[Install the Azure CLI](install-azure-cli.md) or open [Azure Cloud Shell](https://shell.azure.com) to run the scripts in this article.  If you're using a local install of the Azure CLI, version 2.12.0 or later is needed to run `az config param-persist` commands.  Run [az version](/cli/azure/reference-index#az_version) to find the version and dependent libraries that are installed. To upgrade to the latest version, run [az upgrade](/cli/azure/reference-index#az_upgrade).  Azure Cloud Shell always has the latest version of the Azure CLI.
 
 ## Persisted parameter data file
 
-Persisted parameter values are kept in a file named `.param_persist` which is stored in your working directory.  If you are using [Azure Cloud Shell](https://shell.azure.com) to execute Azure CLI commands, your working directory is in the storage account being used by the Azure CLI.  If you are using a [local install](install-azure-cli.md) of the Azure CLI, your working directory is on your local machine.  In either location, the `.param_persist` file is hidden and should not be manually updated.
+Persisted parameter values are kept in a file named `.param_persist` that is stored in your working directory.  If you're using [Azure Cloud Shell](https://shell.azure.com) to execute Azure CLI commands, your working directory is in the storage account being used by the Azure CLI.  If you're using a [local install](install-azure-cli.md) of the Azure CLI, your working directory is on your local machine.  In either location, the `.param_persist` file is hidden and shouldn't be manually updated.
 
 ## Persisted parameter storage and support
 
-The following Azure CLI parameters are supported by persisted parameter.  The `resource_group_name` and `location` parameters are stored differently in that you can add them to persisted parameter _without_ executing a create command.
+No persisted parameters support the following Azure CLI parameters.  The `resource_group_name` and `location` parameters are stored differently in that you can add them to persisted parameter _without_ executing a create command.
```

Comment:
@daphnemamsft , this new sentence makes no sense.  Please rewrite.

Reply:
Sure I have revised this sentence so that it makes sense within the context.

Reply:
What the sentence is saying is that only certain parameters can be used as a persisted parameter.